src/core-styles/mixins/_logical-properties.scss

```diff
@@ -0,0 +1,311 @@
+@import "types";
+@import "validation";
+
+@mixin when-left-to-right {
+  html[dir="ltr"] &:not([dir]),
+  &[dir="ltr"] {
+    @content;
+  }
+}
+
+@mixin when-right-to-left {
+  html[dir="rtl"] &:not([dir]),
+  &[dir="rtl"] {
+    @content;
+  }
+}
+
+@mixin when-logical {
+  html[dir][dir] & {
+    @content;
+  }
+}
```

Comment:
Maybe a conversation for another time, but I think this would make a great stand-alone Sass library. Lot's of reuse potential across other FE projects (incl. Libero)

Reply:
Thanks for this suggestion. It's something we've talked about at bit internally, but there're some wrinkles to iron out before we could consider going down that route. If we do take it further I'll ping you 😄